backend/src/commandTypes.ts

```diff
@@ -74,16 +72,16 @@ export const commandTypes = {
 
   async anyId(value: string, context: CommandContext<any>) {
     const userId = resolveUserId(context.pluginData.client, value);
-    if (userId) return userId;
+    if (userId) return userId as Snowflake;
 
     const channelIdMatch = value.match(channelMentionRegex);
-    if (channelIdMatch) return channelIdMatch[1];
+    if (channelIdMatch) return channelIdMatch[1] as Snowflake;
 
     const roleIdMatch = value.match(roleMentionRegex);
-    if (roleIdMatch) return roleIdMatch[1];
+    if (roleIdMatch) return roleIdMatch[1] as Snowflake;
 
     if (isValidSnowflake(value)) {
```

Comment:
`isValidSnowflake` should return `value is Snowflake` so you don't have to use type assertion afterwards